docs/source/user_guide/getting_started.md

```diff
@@ -0,0 +1,47 @@
+# Getting Started
+
+To get started, first install the `BPX` module using pip
```

Comment:
This conflicts with the installation instructions (which get you do make a venv first). I'm worried users will run `pip install bpx`, then go to installation.md and realise they need to make a venv first

Reply:
good point, I'll just direct people to the installation instructions to avoid having duplicate instructions